eZ/Bundle/EzPublishCoreBundle/Resources/views/content_fields.html.twig

```diff
@@ -427,8 +427,17 @@
         height: parameters.height is defined ? parameters.height : (imageAlias ? imageAlias.height : ''),
         width: parameters.width is defined ? parameters.width : (imageAlias ? imageAlias.width : ''),
     } %}
-
+    {% if parameters.ezlink|default({}) is not empty %}
+    <a
```

Comment:
nitpick: in this file after ```{% if %}``` is indentation

Reply:
@mikadamczyk so something like 0fd888e?

Reply:
👍